test/visual-diff/visual-tests.json

```diff
@@ -9,11 +9,20 @@
     },
     {
       "url": "examples/visual-tests/article.amp/article.amp.html",
-      "name": "AMP Article"
+      "name": "AMP Article",
+      "loading_complete_css": [
+        ".article-body"
+      ]
     },
     {
       "url": "examples/visual-tests/article-access.amp/article-access.amp.html",
-      "name": "AMP Article Access"
+      "name": "AMP Article Access",
+      "loading_complete_css": [
```

Comment:
In the meantime, we can add fields that aren't used by `visual-diff.rb` as a way to document this file. I've done this in the `failing_webpages` section of this file, to note the issue associated with a failure.